lfshttp/cookies.go

```diff
@@ -0,0 +1,20 @@
+package lfshttp
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/google/slothfs/cookie"
+)
+
+func isCookieJarEnabledForHost(c *Client, host string) bool {
+	_, cookieFileOk := c.uc.Get("http", fmt.Sprintf("https://%v", host), "cookieFile")
+
+	return cookieFileOk
+}
+
+func getCookieJarForHost(c *Client, host string) (http.CookieJar, error) {
+cookieFile, _ := c.uc.Get("http", fmt.Sprintf("https://%v", host), "cookieFile")
```

Comment:
This line doesn't seem to be indented properly.  Could you run `goimports` or `go fmt` to make sure your code is tidy?

Reply:
Should be fixed now.